eox_tenant/edxapp_wrapper/backends/enrollments_h_v1.py

```diff
@@ -0,0 +1,11 @@
+""" backend """
+from student import models  # pylint: disable=import-error
+
+
+def get_enrollments_model():
+    """ backend function """
```

Comment:
@cocococosti please improve the description in this docstring.

Reply:
If this PR take us anywhere, doesn't work for me but Jhony is trying to fix it, I'll write better docstrings